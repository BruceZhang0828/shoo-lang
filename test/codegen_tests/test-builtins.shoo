int x = int_of_float(4.5);
float y = float_of_int(4);

if(x==5){
	println("HI");
}

if(y==4.0){
	println("BYE");
}
float result = y + 1.0;

if(result < 6.0){
	println("GOOD");
}

string hello = str_of_float(4.5);
println(hello);

string hello1 = str_of_float(1.245);
println(hello1);