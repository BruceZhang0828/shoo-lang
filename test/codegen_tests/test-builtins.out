HI
BYE
GOOD
4.5
1.245
